CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ x.x.x Release notes (yyyy-MM-dd)
 * Fix an issue where certain `LIKE` queries could hang.
 * Fix an issue where `-[RLMRealm writeCopyToURL:encryptionKey:error]` could create
   a corrupt Realm file.
+* Fix an issue where incrementing a Realm's schema version without actually
```

Comment:
The additive schema mode isn't conceptually tied to sync, but it's currently not exposed as an option for non-sync realms.